database/tests/test_database.py

```diff
@@ -109,13 +109,15 @@ def test_find_biometric_by_id(database, patient_id):
 	result_id = result['id']
 	result_patient_id = result['patient_id']
 	assert str(result_patient_id) == patient_id
-	result_blood_pressure = result['blood_pressure']
+	result_dbp = result['dbp']
+	result_sbp = result['sbp']
 	result_heart_rate = result['heart_rate']
 
 	biometric = find_biometric_by_id(database, result_id)
 	assert biometric['id'] == result_id
 	assert biometric['patient_id'] == result_patient_id
-	assert biometric['blood_pressure'] == result_blood_pressure
```

Comment:
There are still instances of `blood_pressure` in the tests. All these instances should be replaces with sbp and dbp like you did here. Do a find all to find the rest of them

Reply:
Done. Sorry about that.